Patterns nationality

mviegasb opened the pull request:
Nationality pattern built in.

Do NOT merge into master until:
- until this https://github.com/UKHomeOffice/design-system/pull/150 has been pushed live and merged into this branch
- new side nav has been pushed live   
- until it has had an accessibility review

mviegasb commented:
> Format looks good. When asking for multiple nationalities, what does the subsequent screen look like if you add another nationality? This is an example from Register to Vote
> <img alt="Screenshot 2020-06-14 at 15 57 43" width="794" src="https://user-images.githubusercontent.com/13965310/84596889-29cedf80-ae58-11ea-9afc-5d405086e3ba.png">

Ah, yes that's a good idea. In EUSS / Apply it looks like this. Shall I add that to the illustration? @karypun  
![Other country - Apply](https://user-images.githubusercontent.com/62348137/84646509-1ecc8b80-aefa-11ea-9520-fe025b0e7752.png)

mviegasb commented:
I've updated the illustrations and some content tweaks. Ready for a final review. DO NOT PUSH LIVE until this https://github.com/UKHomeOffice/design-system/pull/150 has been pushed live and merged into this branch too.

MarionDownie commented on app/views/patterns/nationality.html:
```diff
@@ -0,0 +1,114 @@
+<!-- RESEARCH -->
+      <h2 class="govuk-heading-m">Research</h2>
+      <p>Further research needs to be done:</p>
+      <ul class="govuk-list govuk-list--bullet">
```
Find these a little hard to read - suggest dashes instead of commas to separate out the clauses. ie 

Further research needs to be done:
* on the best way to ask for a user’s nationality - for example 'What is your country of nationality' or 'What is your nationality?'
* with users who hold multiple nationalities - it can be unclear which nationality they should enter
* around the scenario with users who are stateless - is there alternative information that users can provide?
* on when or when not to ask for country of birth - instead of or as well as nationality 

mviegasb replied:
Thanks! Amended :)

MarionDownie commented on app/views/patterns/nationality.html:
```diff
@@ -0,0 +1,114 @@
+        <li>around the scenario with users who are stateless, is there alternative information that users can provide?</li>
+        <li>on when or when not to ask for country of birth instead of or alongside nationality</li>
+      </ul>
+      <p>We’d also be interested to hear about how services have supported all use cases, for example: country, territories, location, spelling errors and other variations in users answers.</p>
```
Bullets? Also note the apostrophe in the last point.

We’d also be interested to hear about how services have supported all use cases, for example:  
- country
- territories
- location
- spelling errors 
- other variations in users’ answers

mviegasb replied:
Updated

MarionDownie commented on app/views/patterns/nationality.html:
```diff
@@ -0,0 +1,114 @@
+      <p>We’d also be interested to hear about how services have supported all use cases, for example: country, territories, location, spelling errors and other variations in users answers.</p>
+
+<!-- ACCESSIBILLITY -->
+            <h2 class="govuk-heading-m">Accesibility</h2>
```
Typo - Accessibility

mviegasb replied:
Thanks! Amended now

MarionDownie commented on app/views/patterns/nationality.html:
```diff
@@ -0,0 +1,114 @@
+
+<!-- ACCESSIBILLITY -->
+            <h2 class="govuk-heading-m">Accesibility</h2>
+            <p>If your services uses this pattern, we’d be interested to hear any insights you have on any accessibility considerations.</p>
```
If your services use (singular).

I actually tweaked this sentence in the master - can you please check against it?

mviegasb replied:
Have changed it to the one in the master figma file, so: "If your service uses this pattern, let us know of any insights you have on accessibility considerations." Thanks!

mviegasb commented on app/views/patterns/nationality.html:
```diff
@@ -0,0 +1,114 @@
+      <h2 class="govuk-heading-m">Services using this pattern</h2>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>Apply for the EU settlement scheme</li>
+        <li>Generic identity verification</li>
```
Thanks you're right - it's upper case, have amended! (https://www.gov.uk/settled-status-eu-citizens-families)

mviegasb commented:
Nationality pattern is ready for an accessibility review @kitation Thanks!

mviegasb commented:
I've updated the pattern so that it follows the latest updates to the template and merged in the master branch (with the new sidenav). I've also added in a link to the BRP pattern in the side nav. Not sure if there's anything else that needs to be looked at in terms of content? @MarionDownie. Otherwise, all it's missing is an accessibility review @karypun

kitation commented on app/views/patterns/nationality.html:
```diff
@@ -0,0 +1,125 @@
+{% set activesub = "nationality" %}
+
+{% block pageTitle %}
+Patterns template - Home Office design system
```
This needs updating to be the name of the pattern

mviegasb replied:
Thanks, good spot @kitation! Have amended this now. Including on the Brp (biometric residence card or permit details) page too.

mviegasb commented:
@MarionDownie have amended the position of nationality in the side nav, now under 'ask users for...'.

MarionDownie commented:
> @MarionDownie have amended the position of nationality in the side nav, now under 'ask users for...'.

Great, thanks

MarionDownie commented on app/views/patterns/nationality.html:
```diff
@@ -0,0 +1,94 @@
+      <h2 id="research" class="govuk-heading-m">Research</h2>
+      <p class="govuk-body">Further research needs to be done:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>on the best way to ask for a user’s nationality, for example “What is your country of nationality” or “What is your nationality?”</li>
```
Yes, I believe that was the reasoning, and it does stand. I notice the example above from 'register to vote' is different though - did we decide to narrow down our examples?

mviegasb commented:
@karypun @joeedhorton ready to be merged!
